ddtrace/contrib/flask/patch.py

```diff
@@ -70,11 +72,21 @@
 )
 
 
-def get_version():
+def _get_version():
     # type: () -> str
     return get_version_for_package("flask")
 
 
+def get_version():
```

Comment:
We need to make sure that everywhere any of the now deprecated methods are called, we switch them to the non-deprecated private version. E.g. we need to change line 111. This'll fix a lot of the failing tests.